src/components/restaurant/MenuTable.tsx

```diff
@@ -51,7 +65,7 @@ function MenuTable({ title, foodMenu, isActive }) {
               fontSize={{ sm: 16, md: 18, lg: 20 }}
             >
               <UnorderedList spacing={2}>
-                {Object.values(foodMenu[key]).map((dish, index) => (
+                {Object.values(foodMenu[key as FoodMenuKey]).map((dish, index) => (
```

Comment:
not really important but you could raise typing to the outer `(Object.keys(foodMenu) as Array<keyof FoodMenu>).map(...)` having no need to cast later